### PR TITLE
Lab4

### PR DESCRIPTION
**Pull request by ezio-42**

### CODE REVIEW

**Comment by ezio-42** [comment]
@bkmceh @danmory pls review

File: terraform/yandex-cloud/main.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  resources {
+    cores  = var.app_server_cores
+    memory = var.app_server_mem
+  }
+
+  boot_disk {
+    initialize_params {
+      image_id = "fd8c00efhiopj3rlnlbn" # Ubuntu 22.04
```
Comment (danmory) [inline terraform/yandex-cloud/main.tf]: we can put image_id in a variable

Comment (ezio-42) [reply terraform/yandex-cloud/main.tf]: fixed

File: terraform/github/main.tf [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  description = "Innopolis University, DevOps (Fall 2022)"
+  visibility  = "public"
+
+  allow_rebase_merge = false
+  allow_squash_merge = false
+}
+
+resource "github_branch_default" "master" {
+  repository = github_repository.devops-labs.name
+  branch     = "master"
```
Comment (bkmceh) [inline terraform/github/main.tf:23]: What about using variables.tf for that?

**Review by bkmceh — COMMENTED** [review]
Looks good to me except 1 part

File: terraform/github/main.tf (hunk 2 above)
Comment (ezio-42) [reply terraform/github/main.tf:23]: master branch is fixed and will not be changed

**Pull request closed without merging**